### ISSUE TITLE
[V4] Error: Unable to resolve module url from

### ISSUE THREAD

**Issue by andrefelipeschulle** [marklawlor/nativewind#578]
I'm getting this error when configuring v4 in the react-native CLI, I did all the settings for babel, tailwindcss and metro and the problem still persists, has anyone experienced this?

```shell
error: Error: Unable to resolve module url from
 /NativeWind/node_modules/.cache/nativewind/global.css.native.css: 
url could not be found within the project or in these directories:
 node_modules
```

I checked, the .cache folder exists, the global.css.native.css file also exists, the path reaches the file and the file has content inside.

This error occurs when I import the
```
import "./global.css"
```
in App.tsx, and if you commented on the import, it doesn't give an error but a visual effect appears

react-native: v0.72.6
node: v18
nativewind: 4.0.0-alpha.21
OS: Mac OSx - Sonoma 14.0 - M1

**Comment by marklawlor** [comment]
This is impossible for me to debug if no reproduction is shared. Can you please share a link to the repo.

**Comment by andrefelipeschulle** [comment]
@marklawlor Sorry for not having provided more information, if you are using Mac OS, with the M1, follow the link to the project.
https://github.com/andrefelipeschulle/nativewind-test

I configured the test project according to the V4 documentation.
https://www.nativewind.dev/v4/getting-started/metro

the path is valid, the file is created in node_modules
![Captura de Tela 2023-10-14 às 09 33 19](https://github.com/marklawlor/nativewind/assets/33896558/6f5b238b-3c30-4cc3-a9a8-dfd70fa6160f)

more details of the error
![Captura de Tela 2023-10-14 às 09 33 37](https://github.com/marklawlor/nativewind/assets/33896558/1a937c0f-d9d9-4e77-a3ff-4e071b11d1c7)

**Comment by marklawlor** [comment]
Fixed in `4.0.0-alpha.23`
